designer/server/config.ts

```diff
@@ -43,6 +45,8 @@ const schema = joi.object({
   footerText: joi.string().optional(),
   lastCommit: joi.string().default("undefined"),
   lastTag: joi.string().default("undefined"),
+  sessionTimeout: joi.number().default(20 * 60 * 1000),
```

Comment:
whilst it's good to have it in the docs (didn't notice it was missing from there!) I meant in code also - rather than use `20 * 60 * 1000` here, use a variable called `sessionTimeoutInMilliseconds` so there's a closer association to what these numbers are. as it stands we've got the number 1.2 million in code and no context as to what that unit is

Reply:
Done